charts/rstudio-connect/values.yaml

```diff
@@ -104,6 +104,11 @@ pod:
   securityContext: {}
   # -- The containerPort used by the main pod container
   port: 3939
+  # -- Definition for pod lifecycle hooks
```

Comment:
Could you compare this to the `deactivate` function in the `startup.sh` within the default Connect container?

The `deactivate` runs after the Connect process exits; when does this run?